File: examples/petstore/oseg_generated/php/LoginUser_default_example.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPI\Client\Configuration::getDefaultConfiguration();

try {
    $response = (new OpenAPI\Client\Api\UserApi($config))->loginUser(
        username: null,
        password: null,
    );

    print_r($response);
} catch (OpenAPI\Client\ApiException $e) {
    echo 'Exception when calling User#loginUser: '
        . print_r($e->getResponseObject());
}

File: examples/petstore/oseg_generated/csharp/LoginUser_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPITools.Api;
using Org.OpenAPITools.Client;
using Org.OpenAPITools.Model;

public class LoginUserDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            var response = new UserApi(config).LoginUser(
                username: "my_username",
                password: "my_secret_password"
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling User#LoginUser: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
